### PR TITLE
Move beam website to most recent Hugo release (v0.117.0)

### PR DESCRIPTION
**Pull request by lostluck**
Our current hugo version v0.68.3 was released 3+ years ago, and the framework has improved since then. In particular, better error messages would be handy.

The latest release 0.117.0 was released 4 days ago.

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Comment by lostluck** [comment]
http://apache-beam-website-pull-requests.storage.googleapis.com/27974/index.html Navigating around the site, it looks good, so this is proably a safe update.

**Comment by lostluck** [comment]
R: @robertwb 

Just verified and now the error you ran into for the runner guide would have printed the following instead of panicking:
```
Error: error building site: assemble: "/opt/website/www/site/content/en/contribute/runner-guide.md:896:1": failed to extract shortcode: shortcode "highlight" does not evaluate .Inner or .InnerDeindent, yet a closing tag was provided
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
```

File: website/Dockerfile [hunk 1]
```diff
@@ -29,6 +29,8 @@ ENV DEBIAN_FRONTEND=noninteractive \
     LC_CTYPE=C.UTF-8 \
     LC_MESSAGES=C.UTF-8
 
+WORKDIR /opt/
```
Comment (robertwb) [inline website/Dockerfile:32]: Why this change?

**Review by robertwb — APPROVED** [review]
LGTM, both the code and poking around at the generated site. Thanks!

File: website/Dockerfile (hunk 1 above)
Comment (lostluck) [reply website/Dockerfile:32]: Without moving this earlier, npm fails with an otherwise inscrutable "ideal tree" error. This was the reported solution (setting a workdir, so in this case, it's set earlier).
